apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -75,60 +84,122 @@ public class TestDatabaseFixture : IDisposable {
+                                            Key                STRING(MAX) NOT NULL,
+                                            StringValue        STRING(MAX),
+                                          ) PRIMARY KEY (Key)";
             string index1 = "CREATE INDEX TestTableByValue ON TestTable(StringValue)";
             string index2 = "CREATE INDEX TestTableByValueDesc ON TestTable(StringValue DESC)";
 
             // Make the request
             var response =
-                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName),
-                new[] { createTableStatement });
+                await databaseAdminClient.UpdateDatabaseDdlAsync(
+                    new DatabaseName(TestProjectName, TestInstanceName, DatabaseName),
+                    new[] {createTableStatement});
 
             // Poll until the returned long-running operation is complete
             await response.PollUntilCompletedAsync();
 
             response =
-                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName),
-                new[] { index1 });
+                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName,
+                        TestInstanceName,
+                        DatabaseName),
+                    new[] {index1});
             await response.PollUntilCompletedAsync();
 
             response =
-                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName),
-                new[] { index2 });
+                await databaseAdminClient.UpdateDatabaseDdlAsync(new DatabaseName(TestProjectName,
+                        TestInstanceName,
+                        DatabaseName),
+                    new[] {index2});
             await response.PollUntilCompletedAsync();
         }
 
-        public async Task FillSampleData() {
-            using (var connection = new SpannerConnection(ConnectionString)) {
+        public async Task FillSampleData()
+        {
+            using (var connection = new SpannerConnection(ConnectionString))
+            {
                 await connection.OpenAsync();
-                using (var tx = await connection.BeginTransactionAsync()) {
-
+                using (var tx = await connection.BeginTransactionAsync())
+                {
                     var cmd = connection.CreateInsertCommand(TestTable,
-                        new SpannerParameterCollection {
+                        new SpannerParameterCollection
+                        {
                             {"Key", SpannerDbType.String},
                             {"StringValue", SpannerDbType.String}
                         });
                     cmd.Transaction = tx;
 
-                    for (var i = 0; i < 15; ++i) {
+                    for (var i = 0; i < 15; ++i)
```

Comment:
there is only one roundtrip to the server here (during commit).  the execute actually doesnt leave the machine so you'd probably be incurring more of a perf hit by parallelizing these.